dali/base/dautils.hpp

```diff
@@ -541,6 +541,7 @@ extern da_decl ILockInfoCollection *deserializeLockInfoCollection(MemoryBuffer &
 
 extern da_decl IPropertyTreeIterator * getDropZonePlanesIterator(const char * name=nullptr);
 extern da_decl IPropertyTree * getDropZonePlane(const char * name);
+extern da_decl StringBuffer & makeDropZoneScope(StringBuffer & scope, bool isExternalPlane, const char * dropZone, const char * path);
```

Comment:
agree with your comment elsewhere, it would more consistent to use flip this and rename it to isPlane

Reply:
@wangkx - missed this one? i.e. I think worth changing to isPlane so matches other places (e.g. getDropZoneScopePermissions)